src/lib/sample_metadata.rs

```diff
@@ -376,6 +377,13 @@ pub fn coelesce_samples(samples: Vec<SampleMetadata>, lanes: &[usize]) -> Vec<Sa
     // as the order in the input (e.g. sample sheet).
     samples.sort_by_key(|sample| sample.line_number);
 
+    // Update the ordinals
```

Comment:
So we don't get output files with `_S37_` and `_S85_` when the sample is repeated in the sample sheet (but with differing lanes), but instead we get `_S1_` and `_S2_` and so on, in a nice order.

Reply:
Imagine we have Samples 1 through 10, and we specify each four times: once per lane.  Then on the command line, we subset to just lane 2.  Then the ordinals will for sample 2 from lane 2 will be 12, not 2 as we'd like.

Reply:
I suppose we could also have a sample that's just on a particular lane?